### PR TITLE
Task/patand 20 remove unused strings

### PR DESCRIPTION
**Pull request by arlonmukeba**
**Objective**

Remove all unused strings from resource files.

**How To Test**

N/A

Branches

- RS: [task/PATAND-20_Remove_Unused_Strings](https://github.com/Medable/ResearchStack/tree/task/PATAND-20_Remove_Unused_Strings)

### Links

* https://jira.devops.medable.com/browse/PATAND-20

Closes [PATAND-20](https://jira.devops.medable.com/browse/PATAND-20)

### CODE REVIEW

File: backbone/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -32,74 +31,30 @@
...
-    <string name="rsb_consent_share_only">Only share my data with %1$s</string>
-    <string name="rsb_consent_share_description"><![CDATA[%1$s will receive your study data from your
-        participation in this study. \n\nSharing your coded study data more broadly (without
-        information such as your name) may benefit this and future research.<br/><br/><a href="%2$s">
-        Learn more about data sharing</a>]]>
-    </string>
 
     <!-- Consent doc step -->
-    <string name="rsb_consent_review_title">Review</string>
+    <string name="rsb_consent_review_title" tools:ignore="UnusedResources">Review</string>
```
Comment (AbdallahMedable) [inline backbone/src/main/res/values/strings.xml]: Why is this `tools:ignore="UnusedResources"` needed? this string is used in `ConsentTask`

File: backbone/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -32,74 +31,30 @@
     <string name="rsb_consent_section_more_info_withdrawing">Learn more about withdrawing</string>
 
     <!-- Consent sharing step -->
```
Comment (AbdallahMedable) [inline backbone/src/main/res/values/strings.xml]: Please make sure you remove comments that are not needed anymore. This applies to several comments here

File: backbone/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
```
Comment (AbdallahMedable) [inline backbone/src/main/res/values/strings.xml]: If this xmlns is not needed, please remove it

File: backbone/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -143,7 +91,6 @@
 
     <!-- AssetVideoView -->
     <!-- Title for error alert when a video cannot be played.  it can be used by any app. -->
```
Comment (AbdallahMedable) [inline backbone/src/main/res/values/strings.xml]: Remove this comment please

**Comment by AbdallahMedable** [comment]
LGTM, unfortunately I don't have access to approve this.

**Comment by pperotti-medable** [comment]
I see that there are strings that were deleted and were in use. Please review this one by one. The example I found was:    <string name="rsb_consent_dob_full">Date of Birth</string>

This is used in RSConsentConverter

**Comment by Gryzor** [comment]
When I check out this branch in RS and then try build, i get: 

```
> Task :MedableAxon:compileDevDebugJavaWithJavac FAILED
/home/martin/dev/medable/co/test1/MedableAxon/AxonLib/MedableAxon/src/main/java/com/medable/axon/model/researchstack/RSConsentConverter.java:365: error: cannot find symbol
                String firstName = (context != null) ? LocaleUtils.getLocalizedString(context, R.string.rsb_name_first) : "First Name";
                                                                                                       ^
  symbol:   variable rsb_name_first
  location: class string
/home/martin/dev/medable/co/test1/MedableAxon/AxonLib/MedableAxon/src/main/java/com/medable/axon/model/researchstack/RSConsentConverter.java:368: error: cannot find symbol
                String lastName = (context != null) ? LocaleUtils.getLocalizedString(context, R.string.rsb_name_last) : "Last Name";
                                                                                                      ^
  symbol:   variable rsb_name_last
  location: class string
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors

FAILURE: Build failed with an exception.

```

**Pull request closed without merging**